### DISCUSSION TITLE
Dunfell Support for L4T 32.7.1?

### DISCUSSION THREAD

**Discussion by petertalbanese** [OE4T/meta-tegra#946] [General]
Is an update for L4T 32.7.1 coming to the Dunfell branch?

**Comment by ichergui** [comment]
Hey @petertalbanese 
`L4T R-32.7.1` is available in our master branch
We are still testing this release, we will make it available in `dunfell` or Yocto new release.

**Reply by petertalbanese** [reply]
Thanks for the reply!

> dunfell *OR* Yocto new release

Do you happen to know for sure if it will come to `dunfell`? I had heard that it might only go live on the new release of Kirkstone when that comes out. Ideally hoping to stick to `dunfell`, as this is what my other layers are built around.

**Reply by ichergui** [reply]
Yeah, maybe we will make it available in `dunfell` 
Let's hear what @madisongh is thinking

**Reply by madisongh** [reply]
The current plan is to use R32.7.1 for `kirkstone`, the next Yocto Project LTS release.  I am open to back-porting R32.7.1 support to `dunfell` as well, if there is demand for it.

**Reply by ichergui** [reply]
Thanks @madisongh for the updates

**Reply by petertalbanese** [reply]
Got it, thanks for the info! It would definitely be appreciated on my end. Is there a process by which we could gauge overall support for this change?

**Reply by morriskent** [reply]
I'm very interested in dunfell port as well.  I'm not quite ready to move forward just yet.

**Reply by dat-tran-matternet** [reply]
Another vote for dunfell thanks

**Comment by moto-timo** [comment]
Because of the uncertainty of `meta-ros` going forward... (LG Electronics is no longer doing the CI to keep it up to date)... `dunfell` continues to be the release of choice for at least one customer. I am happy to help backport `32.7.1`, but I can only test on `jetson-agx-xavier-devkit`.

**Comment by madisongh** [comment]
I've started the backport and have created a [work-in-progress branch](https://github.com/OE4T/meta-tegra/tree/wip-dunfell-l4t-r32.7.1) for it. There's a [corresponding branch in the demo distro](https://github.com/OE4T/tegra-demo-distro/tree/wip-dunfell-l4t-r32.7.1) for running some test builds.

**Reply by ichergui** [reply]
Perfect 
Thanks @madisongh

**Reply by petertalbanese** [reply]
Much appreciated!

**Comment by madisongh** [comment]
The dunfell branch has been updated to L4T R32.7.1.  There is a `dunfell-l4t-r32.6.1` branch for folks that need to continue using that version of the BSP.

**Reply by petertalbanese** [reply]
Appreciate the quick work on this, thanks!!
